contracts/lib/PercentMath.sol

```diff
@@ -26,6 +26,19 @@ library PercentMath {
         return (_amount * _fracNum) / PCT_DIVISOR;
     }
 
+    /**
+     * @dev Compute percentage that a value represents relative to the principal
+     * @param _amount Amount to calculate the percentage of in principal
+     * @param _principal Amount to calculate the percentage relative to
+     */
+    function inPctOf(uint256 _amount, uint256 _principal)
```

Comment:
I think this function is too coupled to your usecase. What do you think of changing it to a more generic thing?

Reply:
hm, how do you mean we can make it more generic?

Reply:
for instance, not using the word "principal" :p the PercentMath is supposed to be a general purpose

Reply:
oh, I see now :) I assumed it means 'total' in this context since we use that word in my language